web/src/main/react-dashboard/src/components/ProductStatusView.js

```diff
@@ -96,90 +96,93 @@ class ProductStatusView extends Component {
-        </td>
-      </tr>)
-    });
+    render() {
+        const products = this.state.hits.map((product, index) => {
+            return (<tr key={index}>
+                <td> <SingleRecord value={product.productStatus} /> </td>
+                <th onClick={() => this.navigateToRoute( TESTGRID_CONTEXT + "/" + product.productName, {
+                    productId: product.productId,
+                    productName: product.productName,
+                    productStatus :product.productStatus
+                })} scope="row">
+                    <i style={{ cursor: 'pointer' }}>{product.productName}</i>
+                </th>
+                <td style={{ fontSize: '16px' }}>
+                    {(() => {
+                        if (product.lastBuild.modifiedTimestamp) {
+                            return (
+                                <SingleRecord value={product.lastBuild.status}
+                                              nevigate={() => this.navigateToRoute(TESTGRID_CONTEXT + "/" +
+                                                  product.productName, {
+                                                  productId: product.productId,
+                                                  productName: product.productName,
+                                                  productStatus :product.productStatus
+                                              })} time={product.lastBuild.modifiedTimestamp}
+                                />)
+                        } else {
+                            return ("No builds yet!");
+                        }
+                    })()}
+                </td>
+                <td style={{ fontSize: '16px' }}>
+                    {(() => {
+                        if (product.lastfailed.modifiedTimestamp) {
+                            return (
+                                <i onClick={() => this.navigateToRoute(TESTGRID_CONTEXT + "/" + product.productName, {
+                                    productId: product.productId,
+                                    productName: product.productName,
+                                    productStatus :product.productStatus
+                                })} style={{ cursor: 'pointer' }}>
+                                    {Moment(product.lastfailed.modifiedTimestamp).fromNow()}</i>
+                            );
+                        } else {
+                            return ("No failed builds yet!")
+                        }
+                    })()}
+                </td>
+                <td> <img src={require('../play.png')} width="36" height="36" data-tip="Execute job" onClick={() =>
+                { window.location = '/job/'+ product.productName +'/build' }} /> <ReactTooltip /></td>
+                <td ><img src={require('../configure.png')} width="36" height="36" onClick={() => { window.location =
+                    '/job/'+ product.productName +'/configure' }} data-tip="Configure job" style={{ cursor: 'pointer' }}/>
```

Comment:
Is this URL correct? The /job/job-name/configure is coming from Jenkins webapp.

Reply:
Fixed with 12488d2